server/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const app = express();
+const mongoose = require('mongoose');
+const dbIndex = require('../database/index.js');
+const port = 2002;
+
+let database = 'fandangit';
+mongoose.connect(`mongodb://localhost/${database}`, { useNewUrlParser: true });
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  console.log(`connected to ${database}!`);
+});
+
+app.use('/', express.static(__dirname + '/../client/dist')); // verify this path when front end is up
```

Comment:
This all looks great! Have you verified the server is serving up the app as expected, like you wrote on line 16 here?

Reply:
Yes, I need to delete that comment, Thanks!